habit/src/AppScreen.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { createAppContainer } from 'react-navigation';
+
+import { createRootSwitchNavigator } from './AppNavigation';
+import SplashScreen from './containers/SplashScreen';
+
+export default class AppScreen extends React.Component {
+    state = {
+        isLoading: true,
+        isSignedOut: true,
+        userToken: null,
+        isTimeDone: false,
+    };
+
+    componentDidMount() {
+        // eslint-disable-next-line consistent-this
+        let self = this;
```

Comment:
why have you assigned the value of this to self? I think **this** should work fine if you use arrow function in setTimeout